tox.ini

```diff
@@ -90,13 +84,13 @@ commands =
 depends =
 skip_install = true
 deps =
-    coverage<5.0
+    coverage~=5.5
 
 [testenv:end]
 commands =
     coverage report --omit='.tox/*'
     coverage html --omit='.tox/*'
 skip_install = true
-depends = py{35,36,37,38}
+depends = py{35,36,37,38,39}
```

Comment:
This looks like an inconsistency to me. What's the actual minimal supported Python version in unyt atm ? 3.5, 3.6 ?

Reply:
https://github.com/yt-project/unyt/blob/b9c4c21b2e27fd4af9fafa1949dc01296125543c/setup.py#L51

`setup.py` says Python 3.5, but I was going to suggest that you update that to Python 3.6 as you aren't testing it.

Reply:
Can you test it here ? If it still works it seems worth noting, and if it doesn't then we're already past the point where 3.6 is the de facto min requirement, which is also worth knowing :)

Reply:
Yeah. I've tested it before on other branches on my fork and `tox` fails on it. 

```pytb
================================================================================================= ERRORS ==================================================================================================
______________________________________________________________________________________ ERROR collecting test session ______________________________________________________________________________________
Direct construction of DocTestTextfilePlus has been deprecated, please use DocTestTextfilePlus.from_parent.
See https://docs.pytest.org/en/stable/deprecations.html#node-construction-changed-to-node-from-parent for more details.

---------- coverage: platform linux, python 3.5.10-final-0 -----------
Name                               Stmts   Miss  Cover
------------------------------------------------------
unyt/__init__.py                      33     33     0%
unyt/_parsing.py                      38     38     0%
unyt/_physical_ratios.py             111    111     0%
unyt/_pint_conversions.py             16     16     0%
unyt/_unit_lookup_table.py            51     51     0%
unyt/array.py                        708    708     0%
unyt/dimensions.py                    88     88     0%
unyt/equivalencies.py                166    166     0%
unyt/exceptions.py                    66     66     0%
unyt/mpl_interface.py                 76     76     0%
unyt/physical_constants.py             3      3     0%
unyt/testing.py                       14     14     0%
unyt/tests/__init__.py                 0      0   100%
unyt/tests/test_define_unit.py        34     34     0%
unyt/tests/test_mpl_interface.py     175    175     0%
unyt/tests/test_unit_registry.py      84     84     0%
unyt/tests/test_unit_systems.py       75     75     0%
unyt/tests/test_units.py             524    524     0%
unyt/tests/test_unyt_array.py       1608   1608     0%
unyt/tests/test_unyt_testing.py       27     27     0%
unyt/unit_object.py                  464    464     0%
unyt/unit_registry.py                157    157     0%
unyt/unit_symbols.py                   8      8     0%
unyt/unit_systems.py                 159    159     0%
------------------------------------------------------
TOTAL                               4685   4685     0%

========================================================================================= short test summary info =========================================================================================
ERROR 
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================================ 1 error in 0.56s =============================================================================================
ERROR: InvocationError for command /unyt/.tox/py/bin/pytest --cov=unyt --cov-append --doctest-modules --doctest-plus --doctest-rst --basetemp=/unyt/.tox/py/tmp -W once (exited with code 2)
py finish: run-test  after 1.18 seconds
py start: run-test-post 
py finish: run-test-post  after 0.00 seconds
_________________________________________________________________________________________________ summary _________________________________________________________________________________________________
ERROR:   py: commands failed
cleanup /unyt/.tox/.tmp/package/1/unyt-2.8.0+29.g4cb5ec5.dirty.zip
```

Though I just went and in a local `python:3.5` Docker image tested things _without_ `tox` and just running `pytest` manually I can get things to pass.

```console
(venv) root@4bb7b37a5d66:/unyt# python --version --version
Python 3.5.10
(venv) root@4bb7b37a5d66:/unyt# pytest
=========================================================================================== test session starts ===========================================================================================
platform linux -- Python 3.5.10, pytest-6.1.2, py-1.10.0, pluggy-0.13.1
rootdir: /unyt, configfile: setup.cfg
plugins: cov-2.12.1, doctestplus-0.5.0
collected 136 items                                                                                                                                                                                       

unyt/tests/test_define_unit.py ..                                                                                                                                                                   [  1%]
unyt/tests/test_linters.py ..                                                                                                                                                                       [  2%]
unyt/tests/test_mpl_interface.py ...............                                                                                                                                                    [ 13%]
unyt/tests/test_unit_registry.py .......                                                                                                                                                            [ 19%]
unyt/tests/test_unit_systems.py .......                                                                                                                                                             [ 24%]
unyt/tests/test_units.py ................................................                                                                                                                           [ 59%]
unyt/tests/test_unyt_array.py ..................................................                                                                                                                    [ 96%]
unyt/tests/test_unyt_testing.py .....                                                                                                                                                               [100%]

========================================================================================== 136 passed in 11.76s ===========================================================================================
(venv) root@4bb7b37a5d66:/unyt#
```

So I assume that there's some tox problems that I can debug.